FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -13,12 +13,36 @@ public final class RemoteFeedLoader: FeedLoader {
 		case invalidData
 	}
 
+	enum Status: Int {
```

Comment:
The `Status` enum should have a private access level since it is used only internally in the private `map` function.

```suggestion
	private enum Status: Int {
```